libcloud/test/compute/fixtures/openstack_v1.1/_os_volumes.json

```diff
@@ -31,7 +31,7 @@
             "id": "cfcec3bc-b736-4db5-9535-4c24112691b5",
             "metadata": {},
             "size": 50,
-            "snapshotId": null,
+            "snapshotId": "01f48111-7866-4cd2-986a-e92683c4a363",
             "status": "available",
             "volumeType": "None"
         }
```

Comment:
The EC2 fixture already had snapshotId defined. I test this on line 785 of test_ec2.py.